<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Principal</class>
 <widget class="QMainWindow" name="Tela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>910</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>890</width>
      <height>470</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>890</width>
      <height>470</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tela_sobre">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>79</width>
        <height>33</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#0000a3;&quot;&gt;Sobre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>65</y>
        <width>266</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Bla bla bla&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tela_home">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Myriad Pro&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;View/imagens/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>119</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_menu_superior">
     <item>
      <widget class="QPushButton" name="btn_menu_sobre">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Sobre o Sistema&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(44, 153, 255);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(0, 145, 218);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_logof">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Trocar de Usu√°rio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	
	background-color: rgb(51, 51, 51);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_sair">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Sair do Sistema&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(227, 0, 0);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>367</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;View/imagens/icone-header.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Sistema Inova Factory - V 0.1 - By Carlos Eduardo Vieira&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>61</y>
      <width>908</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_menu">
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_menu_home">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_clientes">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Clientes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_servicos">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Servi√ßos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_relatorios">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Relat√≥rios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_financeiro">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Financeiro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu_usuarios">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial,Helvetica,sans-serif</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(220, 220, 220);
	border-radius: 2px;
	font-weight: 500;
	font-size: 12px;
	font-family: Arial, Helvetica, sans-serif;
}

QPushButton:hover{
	
	background-color: rgb(203, 203, 203);
}</string>
       </property>
       <property name="text">
        <string>Usu√°rios</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
